bridges/snowbridge/pallets/system-frontend/src/benchmarking.rs

```diff
@@ -26,8 +26,13 @@ mod benchmarks {
 			decimals: 12,
 		};
 
+		let asset = Asset::from((
+			Location::new(2, [GlobalConsensus(Ethereum { chain_id: 11155111 })]),
+			1000,
+		));
+
```

Comment:
I think you need to use an asset other than Ether, to test the heaviest path. Using Ether skips the asset swap. I did this for the `add_tip` benchmark, you can use the same [helper](https://github.com/paritytech/polkadot-sdk/pull/8725/files#diff-aa37fa51c9a0cc12619d6834e6ed43fb82f944de92d93d36e09537ab6eb35b0fR45).